index.html

```diff
@@ -3,11 +3,200 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="src/css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@1,300&family=League+Spartan:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Adel's to-do app</title>
   </head>
-  <body class="m-0 flex min-h-screen min-w-80 place-items-center">
-    <div class="mx-auto my-0 max-w-7xl p-8 text-center" id="app"></div>
+  <body class="bg-blue-100">
+    <header class="bg-headerimage bg-cover bg-no-repeat">
+      <div class="mx-auto my-0 w-96 px-6 py-12">
+        <section class="mb-8 flex items-center justify-between">
+          <h1 class="text-2xl font-semibold tracking-[7px] text-blue-0">TODO</h1>
```

Comment:
For the tracking, can we use a more responsive unit? See https://tailwindcss.com/docs/letter-spacing for recommended.